src/go/rpk/pkg/cli/cmd/cluster/config/config.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/cluster/config/edit"
```

Comment:
It's not bad to have these in separate subdirectories, but I think since these are all smallish subcommands local to this `config` command, it may be worth it to move these files into a directory local to this one, and then to unexport each of their respective `New` commands. For an example of this approach, see the `group` directory (looking at it myself right now I see that I need to unexport `NewDescribeCommand`).

Reply:
We do lose some encapsulation of any helper functions in the individual command modules by doing this, but there was exactly one of these so I think it's fine.  Did this refactor in a separate commit at end of PR.